Dockerfile.base

```diff
@@ -0,0 +1,9 @@
+FROM elifesciences/sciencebeam-trainer-delft-grobid_unstable:04346b26b5fa7966cef4d6ec6bba867feba52d66-dee3b060495006aa9f8e1b89f6278b44231fe0cc
+
+ENV GROBID_MODELS_DIRECTORY=/opt/grobid/grobid-home/models
```

Comment:
i'm guessin the ENV name with double underscore are based on the base `grobid` appplication and the single underscore are ones you define

Reply:
I suppose it could do with differentiation. The ones with double underscores are overriding configuration properties: https://github.com/kermitt2/grobid/pull/482